packages/ozone/src/daemon/team-profile-synchronizer.ts

```diff
@@ -0,0 +1,17 @@
+import { HOUR } from '@atproto/common'
+import { BackgroundQueue, PeriodicBackgroundTask } from '../background'
+import { AppviewCreator, TeamService } from '../team'
+
+export class TeamProfileSynchronizer extends PeriodicBackgroundTask {
+  constructor(
+    backgroundQueue: BackgroundQueue,
+    appviewCreator: AppviewCreator,
+    teamService: TeamService,
+    interval = 24 * HOUR,
+  ) {
+    super(backgroundQueue, interval, async (_, signal) => {
+      if (signal.aborted) return
```

Comment:
There is no need to check if the signal is aborted here. This check is only needed between tasks (if multiple statements, or during loops)

```suggestion
```